release/release_tests.yaml

```diff
@@ -1801,6 +1801,30 @@
 
   alert: tune_tests
 
+- name: tune_cloud_durable_upload_rllib_str_byod
+  group: Tune cloud tests
+  working_dir: tune_tests/cloud_tests
+  python: "3.8"
```

Comment:
are we moving everything to py38 yet?

Reply:
yes, the list of pip cannot resolve with python 3.7 lol